### PR TITLE
Next Patch

### PR DESCRIPTION
**Pull request by kks110**

### CODE REVIEW

File: command/create_game.rb [hunk 1]
```diff
@@ -43,6 +43,10 @@ def execute(context:)
 
       Stats.create!(player_id: player.id, highest_hp: player.hp, highest_range: player.range)
 
+      if GlobalStats.find_by(player_discord_id: user.id).nil?
+        GlobalStats.create!(player_discord_id: user.id, username: user.username, highest_hp: player.hp, highest_range: player.range)
+      end
```
Comment (AdamC12) [inline command/create_game.rb:48]: Does it create most_cities_hsld when it does the check in capture_city?

Comment (AdamC12) [reply command/create_game.rb:48]: Same with energy given

Comment (kks110) [reply command/create_game.rb:48]: Yes and yes :)

Comment (AdamC12) [reply command/create_game.rb:48]: ✌️

**Pull request merged**